manifests/broker.pp

```diff
@@ -0,0 +1,20 @@
+# Set up the broker
+class pulp::broker {
+  if $pulp::manage_broker {
```

Comment:
Should we make this a parameter of this class? Same question for the 'messaging_transport' parameter? The reason I ask is this looks fairly independent and might could be used as an independent class to establish broker services? Granted, on the Katello side we do not currently do interesting Pulp setups at present outside of master and nodes.